### DISCUSSION TITLE
Software sound settings dont work

### DISCUSSION THREAD

**Discussion by Flaiks123** [MiczFlor/RPi-Jukebox-RFID#1063] [Q&A]
Hey @ all

First: I'm an absolute noob/beginner with raspberry pi and all this stuff. Please help me with "easy language", step by step :) Thanks.

Setup a fresh phoniebox:

- fresh buster
- phoniebox classic, without spotify
- all stuff was working, music, panel ..
  .. only the software sound controls in the online panel dont work.

I also try to go into mpd.conf and change the **mixer typ** from **hardware** to **software**, but also it doesnt work.

Anyone can help me to solve this little problem? :) 

Greetings!

**Comment by s-martin** [comment]
Hi, what’s the result of `tail -n 500 /var/log/syslog`?
